packages/neuron-ui/src/utils/scriptAndAddress.ts

```diff
@@ -21,3 +24,22 @@ export const addressToScript = (address: string, { isMainnet = true }: { isMainn
   const config = CONFIGS[prefix] ?? (isMainnet ? MAINNET : TESTNET)
   return parseAddress(address, { config })
 }
+
+export const toLongAddress = (address: string) => {
+  try {
+    return scriptToAddress(addressToScript(address), { isMainnet: address.startsWith(MAINNET.PREFIX) })
+  } catch {
+    return ''
+  }
+}
+
+export const toShortAddress = (address: string) => {
+  try {
+    return scriptToAddress(addressToScript(address), {
+      isMainnet: address.startsWith(MAINNET.PREFIX),
+      deprecated: true,
+    })
+  } catch {
+    return ''
+  }
+}
```

Comment:
These 2 functions can be done in 1 function with a parameter `deprecated?: boolean`

Reply:
Combine `toLongAddress()` and `toShortAddress()` into `addressToAddress()`
```ts
export const addressToAddress = (address: string, { deprecated = false }: { deprecated?: boolean } = {}) => {
  try {
    return scriptToAddress(addressToScript(address), {
      isMainnet: address.startsWith(MAINNET.PREFIX),
      deprecated,
    })
  } catch {
    return ''
  }
}
```